src/Sentry/DynamicSamplingContext.cs

```diff
@@ -61,6 +68,11 @@ private DynamicSamplingContext(
             items.Add("sample_rate", sampleRate.Value.ToString(CultureInfo.InvariantCulture));
         }
 
+        if (sampleRand is not null)
+        {
+            items.Add("sample_rand", sampleRand.Value.ToString("N4", CultureInfo.InvariantCulture));
```

Comment:
we want this even if we have `sample_rate` in? see line 68

Reply:
We do. This is strictly required for the DSC to be valid: https://develop.sentry.dev/sdk/telemetry/traces/dynamic-sampling-context/#dsc-specification

Reply:
I believe so yes. The docs developer docs indicate:
> If sample_rate (inside baggage) and the sampling decision (trailing -1 or -0 from the sentry-trace header) are present in the incoming trace, create sample_rand so that when compared to the incoming sample_rate it would lead to the same sampling decision that is in the sentry-trace header. 

Which we're doing... and then we propagate the `sample_rand` so that if any downstream nodes can use this in their sampling decision.